### DISCUSSION TITLE
Pan gesture to zoom in and out

### DISCUSSION THREAD

**Discussion by arieldiporto95** [buildwiththeta/buildwiththeta#103] [Ideas]
I think can be useful to add the pan gesture to zoom in and out in while editing the pages.

**Comment by spioc999** [comment]
I agree on that!
I would propose to add this feature on the main view.

![Screenshot 2023-06-09 at 16 21 21](https://github.com/buildwiththeta/buildwiththeta/assets/52169960/9e8a2a03-062e-4c95-b6f4-63bd6c9b694e)

In this situation, it seems very difficult to select and work with components and it's not able to scroll too

**Comment by andreabuttarelli** [comment]
We will add a zoom indicator to zoom in and out of the device screen 💫
